ajax/libs/vega/package.json

```diff
@@ -13,8 +13,13 @@
     {
       "basePath": "",
       "files": [
-        "vega.js",
-        "vega.min.js"
+        "vega*.(js|css|map)"
+      ]
+    },
+    {
+      "basePath": "build",
+      "files": [
+        "vega*.(js|css|map)"
```

Comment:
As we discussed earlier, maybe you'll like to use `"**/vega*.(js|css|map)"` here?